### ISSUE TITLE
Customize URL websocket connection

### ISSUE THREAD

**Issue by maulikmadhavi** [Kaljurand/K6nele#70]
Dear All

Our URL contains IP, port and token all three together.
Handle result shows {"status": 5, "message": "Could not determine type of stream.", "id": "4ae116fe-7cb7-49e5-93c1-b4d5ca07b9bd"}

Unable to connect websocket and UI show No transcription found

Please help me to connect the websocket.

Many thanks

**Comment by Kaljurand** [comment]
What do you mean by "token"?

The server URL should look something like this

```
ws://10.0.0.11:8080/client/ws/speech?content-type=audio/x-flac&lang=et-EE&user-agent=...&calling-package=...&user-id=...&partial=true
```

where the ``ws://10.0.0.11:8080/client/ws/speech`` part can be configured in the service settings or set using the ``EXTRA_SERVER_URL`` extra. This part cannot contain "?" nor any additional query params. (Is that the issue?)

**Comment by Kaljurand** [comment]
Considering it fixed by 29717b0a7 (v1.7.10). Please reopen if the issue remains.
